### PR TITLE
Feat chairs

### PR DESCRIPTION
**Pull request by mixcas**
Add chairs basic functionality
Test images are in the sync
**NOTE**: Code comes with some `vars` enabled for developing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -62,9 +62,13 @@ <h1 class="u-visuallyhidden">Seatporn</h1>
...
-      <main id="main-content" class="grid-row justify-center align-items-center">
-        <div id="image-holder" class="grid-row justify-center align-items-center">
-          <img src="img/dist/chair.jpeg">
+      <main id="main-content">
+        <div id="images-container" class="grid-row justify-center align-items-center">
+          <!--
+          <div class="image-holder grid-row justify-center align-items-center">
+            <img src="chairs/0079.jpeg">
+          </div>
+          -->
```
Comment (mikeyrayvon) [inline index.html]: probably can remove this commented markup right?

File: js/main.js [hunk 2]
```diff
@@ -27,6 +28,162 @@ Site = {
...
+
+    for(var i = 0; i <= _this.tailSize; i++) {
+      _this.tail[i] = _this.list[i + _this.currentPosition];
+    }
+  },
+
+  loadImages: function() {
+    var _this = this;
+
+    for(var image of _this.tail) {
```
Comment (varffs) [inline js/main.js]: do we have this being complied into old school js?

File: js/main.js [hunk 3]
```diff
@@ -27,6 +28,162 @@ Site = {
...
+
+    return _this.padNumber(base) + ".jpeg";
+  },
+
+  generateImage: function(base) {
+    var _this = this;
+
+    var filename = _this.formatName(base);
+    var imageHolder = document.createElement('div');
+    var image = document.createElement('img');
```
Comment (varffs) [inline js/main.js:150]: is creating 2 new elements quicker than DOM inserting markup?

File: js/main.js [hunk 4]
```diff
@@ -27,6 +28,162 @@ Site = {
...
+
+    imageHolder.appendChild(image);
+
+    return imageHolder;
+  },
+
+  triggerTimer: function() {
+    var _this = this;
+
+    setTimeout(function() {
```
Comment (varffs) [inline js/main.js:164]: are you using the timer lib?

File: index.html (hunk 1 above)
Comment (varffs) [reply index.html]: agree

**Review by varffs — COMMENTED** [review]
Any value in a Site var for debug so we don't switch commented out js?

File: css/site.styl [hunk 5]
```diff
@@ -76,17 +78,9 @@ img
...
   pointer-events: none
-
-@keyframes hide-loading
-  0%
-    opacity: 1
-  80%
-    opacity: 1
-  100%
-    opacity: 0
+  transition: opacity 1s ease-in-out
```
Comment (mikeyrayvon) [inline css/site.styl]: also should add `will-change: opacity` here

File: css/site.styl [hunk 6]
```diff
@@ -97,6 +91,7 @@ img
     stroke-dasharray: 1000
     animation: draw 4s linear infinite
     animation-fill-mode: forwards
+  transition: opacity 1s ease-in-out
```
Comment (mikeyrayvon) [inline css/site.styl]: `will-change: opacity` here as well

File: js/main.js (hunk 3 above)
Comment (mixcas) [reply js/main.js:150]: idk

File: js/main.js (hunk 4 above)
Comment (mixcas) [reply js/main.js:164]: oh no, forgot to remove

File: js/main.js (hunk 2 above)
Comment (mixcas) [reply js/main.js]: good question. no. i rather rewrite this part than adding transpiling to the project.

File: index.html (hunk 1 above)
Comment (mixcas) [reply index.html]: yep

**Comment by mixcas** [comment]
Ready

**Pull request merged**